lua/sp.c

```diff
@@ -7182,6 +7182,9 @@ static int exec_procedure_int(struct sqlthdstate *thd,
         rc = push_args_and_run_sp(clnt, end_ptr, err);
     }
 
+    if (trigger || consumer)
```

Comment:
Isn't this too late? The procedure is done running when `push_args_and_run_sp` returns above.

Reply:
done, I tested it using https://github.com/bloomberg/comdb2/commit/3977356b9cc5399adc92365c32ff57473f361d27 before